PizzaOvenUI/Screen_MainMenu.qml

```diff
@@ -45,24 +45,30 @@ Item {
         setMenuPositionFromFoodIndex();
     }
 
+    function loadTheFoodListIfEmpty() {
+        if (foodListModel.count == 0) {
```

Comment:
This line is the real change. I decided to move the following lines into this descriptive function name to make it a little more clear.